flow/carrier_service/create.yaml

```
@@ -0,0 +1,20 @@
+triggers:
+  - keyword: create_shopify_carrier_service
+
+steps:
+  - ask: What name you want for your carrier service
+  - flow_set: carrier_service_name
+  - ask: >
+      So, do you want your carrier service
+      to be named (@ flow.carrier_service_name )?
+    quick_replies:
+      - text: No
+        action:
+          flow: flow.start
+          transfer: true
```

It would look a bit odd to have just a "No" quick reply and expect "Yes" (text not checked) via the composer

```suggestion
  - ask: What name you want for your carrier service?
  - flow_set: carrier_service_name
```

---

Not sure what you mean by text not checked. Yes is the case when the text checked, right?

---

I mean if you type "yes" or "no" or "whatever", the flow doesn't change. You need to type something but it doesn't matter, so it'll be a weird experience.

If you still want to get the user to confirm the name, use two buttons (not quick replies), for Yes and No.